integration-test/src/types/import-test.ts

```diff
@@ -31,7 +31,6 @@ import {
   ReservedLineweight,
   Measurement,
   ObscuredLineTypes,
-  INDEXED_CHUNK_SIZE,
```

Comment:
This is not a constant related to dxf parser.

Fun fact! The origin of `dxf-json` was [`dxf-viewer`](https://github.com/vagran/dxf-viewer) and [`dxf-parser`](https://github.com/gdsestimating/dxf-parser), although currently there is no common source code anymore. This constant was from there if I remeber right.